### PR TITLE
Pauld/add support for backup storage account

### PR DESCRIPTION
**Pull request by pauld-msft**
Adds to the copy prod sdks to destination job to make it run weekly to copy from prod to the prod backup storage account

### CODE REVIEW

File: doc/supportedPlatformVersions.md [hunk 1]
```diff
@@ -398,6 +398,9 @@
 - 16.13.2
 - 16.14.0
 - 16.14.2
+- 18.0.0
+- 18.1.0
+- 18.2.0
```
Comment (pauld-msft) [inline doc/supportedPlatformVersions.md:403]: unrelated to this change, looks like node 18 was incorrectly placed under the stretch versions and running the generate script fixed it

File: vsts/pipelines/copySdksFromProdToStorageAccount.yml [hunk 2]
```diff
@@ -37,10 +44,18 @@ jobs:
...
 
 trigger: none
 pr: none
+schedules:
+- cron: "0 12 * * 0"
+  displayName: Weekly Sunday Copy to Backup Prod
+  branches:
+    include:
+    - main
+  always: true # whether to always run the pipeline even if there have been source code changes since the last run
```
Comment (pauld-msft) [inline vsts/pipelines/copySdksFromProdToStorageAccount.yml:61]: Kind of a bummer, but couldn't find a way to pass in parameters for this run, so needed to update the defaults of the job to be what we want to run on the schedule

**Review by daniv-msft — APPROVED** [review]
Looks good to me

**Review by cormacpayne — APPROVED** [review]
:shipit:

**Comment by william-msft** [comment]
Re-ran validation build failed jobs

**Review by william-msft — APPROVED** [review]

**Pull request merged**